main.js

```diff
@@ -62,6 +63,20 @@ process.on("SIGUSR2", () => {
 });
 
 crawler = new Crawler();
-crawler.run();
 
+const app = express();
+const port = 3000;
+console.log("crawler created");
+app.get("/", (req, res) => {
```

Comment:
I wonder if this should be a POST instead? Do you really want to trigger a new crawl any time someone does a GET to this URL?